commit 527ef0550d79e3b3a0ef8f5061072075afef6aaf
Author: Arnaud Patard <arnaud.patard@rtp-net.org>
Date:   Thu Dec 1 11:58:25 2011 +0100

    ARM: Kirkwood: Add configuration for MPP12 as GPIO
    
    The MPP12 is listed in the 6281 HW manual as output only but the iconnect
    board from iomega is using it as GPIO (there's a button connected on it). So,
    I'm adding a definition for the MPP12 as GPIO. As I've no informations about
    this and which kirkwood are "affected", I'm adding a new #define instead of
    modifying the current one for MPP12.
    
    Signed-off-by: Arnaud Patard <arnaud.patard@rtp-net.org>
    Signed-off-by: Nicolas Pitre <nico@fluxnic.net>

Index: sid/arch/arm/mach-kirkwood/mpp.h
===================================================================
--- sid.orig/arch/arm/mach-kirkwood/mpp.h	2012-05-31 01:44:12.000000000 +0200
+++ sid/arch/arm/mach-kirkwood/mpp.h	2012-06-10 10:18:11.502678583 +0200
@@ -102,6 +102,7 @@
 #define MPP11_SATA0_ACTn	MPP( 11, 0x5, 0, 0, 0,   1,   1,   1,   1 )
 
 #define MPP12_GPO		MPP( 12, 0x0, 0, 1, 1,   1,   1,   1,   1 )
+#define MPP12_GPIO		MPP( 12, 0x0, 1, 1, 0,   0,   0,   1,   0 )
 #define MPP12_SD_CLK		MPP( 12, 0x1, 0, 0, 1,   1,   1,   1,   1 )
 #define MPP12_AU_SPDIF0		MPP( 12, 0xa, 0, 0, 0,   0,   0,   0,   1 )
 #define MPP12_SPI_MOSI		MPP( 12, 0xb, 0, 0, 0,   0,   0,   0,   1 )
